.github/workflows/deploy_tests.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           pip install -e . --group coverage
 
       - name: Run tests
-        run: python -m pytest -vv --cov=nwb2bids --cov-report xml:./codecov.xml
+        run: python -m pytest -m "not remote" -vv --cov=nwb2bids --cov-report xml:./codecov.xml
```

Comment:
The tests marked with `@pytest.mark.remote` are intentionally deselected here, but they are not run in any other workflow either. I think this is the reason why the coverage is 25% as @yarikoptic pointed out at https://github.com/con/nwb2bids/pull/30#pullrequestreview-3218805987.

Additionally, not related to the changes in this PR, I would suggest restricting the test discovery to the `tests` subfolder.